malformed/test_purge_hch.py

```diff
@@ -0,0 +1,148 @@
+from framework import tester
+from helpers import tf_cfg, deproxy, tempesta
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2021 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class HeavyChunkedPurgeTest(tester.TempestaTest):
+    # This is a heavy chunked test for ss_skb_chop_head_tail() function in context
+    # of rewriting PURGE method as GET, issue #1535
+    #
+    # TODO: move the test to appropriate place (/cache/?)
```

Comment:
Yes, it'd be good to move the test to `/cache`

Reply:
Done